src/Microsoft.AspNetCore.SystemWebAdapters.SessionState/Microsoft.AspNetCore.SystemWebAdapters.SessionState.csproj

```diff
@@ -11,18 +10,13 @@
 
   <ItemGroup>
     <InternalsVisibleTo Include="Microsoft.AspNetCore.SystemWebAdapters.SessionState.Tests" />
+    <InternalsVisibleTo Include="Microsoft.AspNetCore.SystemWebAdapters.SessionState.Framework.Tests" />
     <InternalsVisibleTo Include="DynamicProxyGenAssembly2" Key="0024000004800000940000000602000000240000525341310004000001000100c547cac37abd99c8db225ef2f6c8a3602f3b3606cc9891605d02baa56104f4cfc0734aa39b93bf7852f7d9266654753cc297e7d2edfe0bac1cdcf9f717241550e0a7b191195b7667bb4f64bcb8e2121380fd1d9d46ad2d92d2d15605093924cceaf74c4861eff62abf69b9291ed0a340e113be11e6a7d3113e92484cf7045cc7" />
   </ItemGroup>
 
   <ItemGroup>
     <FrameworkFiles Include="**/*.Framework.cs" />
-    <StandardFiles Include="**/*.Standard.cs" />
-  </ItemGroup>
-
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
-    <Compile Remove="**/*" />
-    <Compile Include="@(StandardFiles)" />
-    <Compile Include="RemoteSession/RemoteAppSessionStateOptions.cs" />
+    <FrameworkFiles Include="RemoteSession/Framework/*.cs" />
```

Comment:
Do we want both `*.Framework.cs` and `/Framework/*.cs` denoting netfx-only source? It might be cleaner to just use one or the other.

Reply:
I have no strong opinion here and am switching to the `Framework/*.cs` as there's enough files.